### PR TITLE
Pn 46 fe set up navigation bar

### PR DESCRIPTION
**Pull request by applestory**

### CODE REVIEW

**Comment by ShawnWang86b** [comment]
need to responsive design and unit test

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,8 +1,114 @@
...
+      )}
+
+      {isLoggedIn && (
+        <HStack height="100%" spacing="1em">
+          <HStack height="100%" spacing="0.5em">
+            <Menu>
+              <Avatar name="M" src="" bg="gray" color="white" />
+              <MenuButton
+                as={Button}
+                rightIcon={<ChevronDownIcon />}
```
Comment (ShawnWang86b) [inline src/components/Navbar.tsx:69]: you can add <ChevronUpIcon /> when the Menu opened

File: src/components/Navbar.tsx [hunk 2]
```diff
@@ -1,8 +1,114 @@
...
+                height="80%"
+                padding="0em"
+              >
+                {`Mark  `}
+
+                <Icon viewBox="0 0 200 200" color="red.500">
+                  <path
+                    fill="currentColor"
+                    d="M 100, 100 m -75, 0 a 75,75 0 1,0 150,0 a 75,75 0 1,0 -150,0"
+                  />
```
Comment (ShawnWang86b) [inline src/components/Navbar.tsx:80]: the red point icon is better to store in 'assets' folder in the src, so far seems we don't have this folder in repo, you can create one

File: src/components/Navbar.tsx [hunk 3]
```diff
@@ -1,8 +1,114 @@
...
 
 const Navbar = () => {
-  return <div>Navbar</div>;
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  function changeIsLoggedStatus() {
+    setIsLoggedIn((prevIsLoggedStatus: boolean) => {
+      return !prevIsLoggedStatus;
+    });
+  }
+
```
Comment (ShawnWang86b) [inline src/components/Navbar.tsx:28]: const [isLoggedIn, setIsLoggedIn] = useState(false) and  changeIsLoggedStatus() can change to :
const [user, setUser] = state('')
 when login setUser = ('someone')
{user&& <logined state component /> ... }
when log out {setUser(' ')}
but considering we don't finish the login part, you can do it in the later sprint
you can start to study how to get the data by using redux.

File: src/components/Navbar.tsx (hunk 1 above)
Comment (ShawnWang86b) [reply src/components/Navbar.tsx:69]: ChevronUpIcon

**Comment by jube33717f** [comment]
After updating your code, please remove the 'changes request' label on the right and add 'can review' label👍

**Review by ShawnWang86b — DISMISSED** [review]

**Review by ShawnWang86b — APPROVED** [review]

**Comment by applestory** [comment]
Thanks for reviewing, Everyone!

**Pull request merged**